src/components/resources/MyResources.js

```diff
@@ -0,0 +1,514 @@
+            method: "DELETE",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                resourceID: selectedResource.id
+            })
+        });
+        console.log("want to delete resource: " + selectedResource.id)
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    const handleOpenDialog = (resource) => {
+        setSelectedResource(resource);
+        console.log(selectedResource)
+        setIsDialogOpen(true);
+
+        // Update resource title
+        setResourceTitle(resource.resourcesTitle);
+        setResourceTitleError(false);
+        setResourceTitleErrorText('');
+
+        // Update resource link
+        setResourceLink(resource.resourcesLink);
+        setResourceLinkError(false);
+        setResourceLinkErrorText('');
+
+        // Update resource media tag
+        setMediaTag(resource.mediaTag);
+    };
+
+    const handleOpenDeleteDialog = (resource) => {
+        setSelectedResource(resource);
+        setOpenDeleteDialog(true);
+    }
+
+    const handleCloseDeleteDialog = () => {
+        setSelectedResource(null);
+        setOpenDeleteDialog(false);
+    }
+
+    const handleCloseDialog = () => {
+        setSelectedResource(null);
+        setIsDialogOpen(false);
+    };
+
+    const handleRefreshFilter = async () => {
+        setResourceTag("");
+    }
+
+    const handleRefreshSearch = async () => {
+        setSearchTerm("");
+        setRefreshSearch(refreshSearch + 1);
+    }
+
+    return (
+        <div id="body">
+
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 115, left: '50%', transform: 'translate(-50%, -50%)' }}>
+                <Typography
+                    variant="h4"
+                    gutterBottom
+                    component="div">
+                    My Resources
+                </Typography>
+            </Box>
+
+            <Box sx={{ width: '600px', position: 'absolute', top: 185, left: '50%', transform: 'translateX(-50%)', marginBottom: '20px' }}>
+                <Search
+                    label="Search for resource names"
+                    searchTerm={searchTerm}
+                    onSetSearch={handleSearch}
+                    fullWidth
+                    onButtonClick={loadgetResourcesByUser}
+                    onResetSearch={handleRefreshSearch}
+                />
+                <br />
+                <Typography
+                    style={{ color: "black", mb: 2, fontSize: 14, align: 'right' }}
+                >
+                    Filters
+                </Typography>
+                <DropdownFilter
+                    placeholder="Select a Media Type Tag"
+                    value={resourceTag}
+                    onChange={handleResourceTag}
+                    lists={mediaTagList}
```

Comment:
Might have to create a new list without the extra space at the start... not quite sure sorry!

Reply:
Yeah, I added a new list without the blank tag from the filter